### DISCUSSION TITLE
undefined symbol: _ZN3c1017RegisterOperatorsD1Ev

### DISCUSSION THREAD

**Discussion by RX28666** [pyg-team/pytorch_geometric#8962] [Q&A]
Hello,

I just update my pytorch to 2.2.1+cu121 using pip, and install corresponding pyg by `pip install torch_geometric`, now it returns 
`lib/python3.10/site-packages/torch_cluster/_version_cuda.so: undefined symbol: _ZN3c1017RegisterOperatorsD1Ev`, which didn't happen for my previous version pytorch and pyg. I have found the discussion: `https://github.com/pyg-team/pytorch_geometric/issues/999`, but I am still confused about how to solve it. Any help would be appreciated.

**Comment by ericbenoit74** [comment]
Hello,

I got got the same issue today when installing the stable version of TensorRT-LLM, and solved it by installing the preview version instead. It seems that the issue was solved by downgrading several packages, including torch (2.2.1 -> 2.1.2) and cu12 (2.19.3 -> 2.18.1)
